core/shared/src/main/scala/zio/Fiber.scala

```diff
@@ -416,7 +416,7 @@ object Fiber {
    * A synthetic fiber that is created from a pure value or that combines
    * existing fibers.
    */
-  sealed trait Synthetic[+E, +A] extends Fiber[E, A] {}
+  private[zio] abstract class Synthetic[+E, +A] extends Fiber[E, A] {}
```

Comment:
This needs to be sealed, ideally. There is an internal object you can use to extend `Runtime` (from within the `zio` package), and we can do the same trick for `Synthetic`. /cc @adamgfraser

Reply:
Oh, nice 👍